### ISSUE TITLE
Merge records settings

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1204]
Will so great if will in SuiteCRM some tool, where you can set up which the merge fields will use module for merge function, because i am sure that many companies have many custom fields and it's complicated (you must do it manualy) merge 2 or more records with many custom fields.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Merge records settings](https://trello.com/c/Od7PJw3s/108-merge-records-settings)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
